cli.js

```diff
@@ -1,53 +1,129 @@
 #!/usr/bin/env node
-import fs from 'node:fs'
 import { globbySync } from 'globby'
+import fs from 'node:fs'
+import { parseArgs } from 'node:util'
```

Comment:
I believe we still support Node.js v12, https://github.com/keithamus/sort-package-json/blob/dc1ad5a340d1b03ff35d39f267f0b7a827b1a06f/.github/workflows/pr.yml#L17 this is not available.

Reply:
I would love a pointer on how to resolve this. Do I need to find a polyfill, or a different module, or is there some better way to include the module that works with older versions?

Reply:
I'd suggest to postpone this change before we drop these old versions. It's refactor, not changing the program behavior.

Reply:
do we have a road map for when the old versions might be dropped?

Reply:
It's common to follow the Node.js release schedule. https://github.com/nodejs/release#release-schedule

Reply:
if i'm reading this right, we're looking at October for 1.16.x to be discontinued? is that correct?

Reply:
[`parseArgs`](
https://nodejs.org/api/util.html#utilparseargsconfig) is available in Node.js v 16, so 2023-04-30(EOL of v14) should be fine. 

Like I said it's a refactor so it can wait.

Reply:
alternatively, we *could* use this [parseargs polyfill](https://github.com/pkgjs/parseargs) to release this now and then change the import to the node:util version when 14 is discontinued

Reply:
Yes we can, but I don't think worth to introduce a new dependency just for this.

Why don't you focus on adding `--quite` for now? Simple array filter should good enough.